.travis/run-unit-tests.sh

```diff
@@ -79,7 +79,7 @@ if [ -z ${run_c_unit_tests} ] ; then
 else
     install_netdata || exit 1
 
-    if [ -n ${run_c_unit_tests} ] ; then
+    if [ -n "${run_c_unit_tests}" ] ; then
```

Comment:
Can we omit the space preceding ; as we do in other places?

```suggestion
    if [ -n "${run_c_unit_tests}" ]; then
```